tests/test_py_kernel.py

```diff
@@ -13,6 +13,9 @@ def execute(self, input_columns):
         point = protobufs.Point()
         point.x = 10
         point.y = 5
-        return [point.SerializeToString()]
+        input_count = len(input_columns[0])
+        column_count = len(input_columns)
+        return [[point.SerializeToString() for _ in xrange(input_count)]
+                for _ in xrange(column_count)]
```

Comment:
One behavior that might be useful is if the kernel does not batch, then it should be able to just return a 1d list (one entry for each column) as before. This would just require the C++ wrapper kernel to condition on `can_batch` to handle the unbatched case.